src/urllib3/util/ssl_.py

```diff
@@ -223,7 +230,13 @@ def create_urllib3_context(
     """
     if SSLContext is None:
         raise TypeError("Can't create an SSLContext object without an ssl module")
-    context = SSLContext(ssl_version or PROTOCOL_TLS)
+
+    # PROTOCOL_TLS is deprecated in Python 3.10 so we pass PROTOCOL_TLS_CLIENT instead.
+    # If it's not available in 'ssl' the value is aliased to PROTOCOL_TLS anyways.
+    if ssl_version == PROTOCOL_TLS:
+        ssl_version = PROTOCOL_TLS_CLIENT
```

Comment:
In fact all ``PROTOCOL_*`` except for ``PROTOCOL_TLS_CLIENT`` and ``PROTOCOL_TLS_SERVER`` are deprecated. OpenSSL has deprecated the other methods, too.